flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -1904,6 +2055,25 @@ ANY, and(logical(LogicalTypeRoot.BOOLEAN), LITERAL)
     public static final BuiltInFunctionDefinition GET =
             BuiltInFunctionDefinition.newBuilder()
                     .name("get")
+                    .callSyntax(
+                            (sqlName, operands) -> {
+                                final Optional<String> fieldName =
+                                        ((ValueLiteralExpression) operands.get(1))
+                                                .getValueAs(String.class);
+
+                                return fieldName
+                                        .map(
+                                                n ->
+                                                        String.format(
+                                                                "%s.`%s`",
```

Comment:
not sure if the serialization format is correct here:
```
`%s`

```
 shouldn't this be properly escaped in the argument already